zerver/webhooks/transifex/tests.py

```diff
@@ -25,7 +25,7 @@ def test_transifex_reviewed_message(self):
             language=self.LANGUAGE,
             resource=self.RESOURCE,
         )
-        self.send_and_test_stream_message(None, expected_subject, expected_message)
+        self.send_and_test_stream_message(None, expected_subject, expected_message) # type: ignore
```

Comment:
Why are we ignoring this? I think passing `None` as the first parameter should throw an error, but I don't know why it doesn't.